tests/spelling.R

```diff
@@ -1,6 +1,5 @@
-if (requireNamespace("spelling", quietly = TRUE)) {
-  spelling::spell_check_test(
-    vignettes = TRUE, error = FALSE,
-    skip_on_cran = TRUE
-  )
-}
+rlang::check_installed("spelling")
+spelling::spell_check_test(
+  vignettes = TRUE, error = FALSE,
+  skip_on_cran = TRUE
+)
```

Comment:
I don't believe this is correct because the `if` statement is now gone so the code on line 2 would execute not in relation to the code that checks if the package is installed. Happy to chat more if my understanding is incorrect.

Reply:
I could do:
```
if (rlang::check_installed("spelling")) {
spelling::spell_check_test(
  vignettes = TRUE, error = FALSE,
  skip_on_cran = TRUE
)
}
```
But I don't think it's necessary unless we don't require the user to have `spelling` installed and are only spell checking if they do have `spelling` installed. Otherwise, rlang should check for `spelling`, download it if it isn't already installed, and then run the spell check.

Reply:
I think it is fine as is, I tested and get the same result with the update to the code you have made. Closing this as resolved.